src/main/app/src/app/zaken/intake-afronden-dialog/intake-afronden-dialog.component.ts

```diff
@@ -123,20 +109,23 @@ export class IntakeAfrondenDialogComponent implements OnInit, OnDestroy {
         userEventListenerData.zaakOntvankelijk = this.radioFormField.formControl.value.value;
         userEventListenerData.resultaatToelichting = this.redenFormField.formControl.value;
 
-        let onderwerp = userEventListenerData.zaakOntvankelijk ? this.onderwerpOntvankelijk : this.onderwerpNietOntvankelijk;
-        onderwerp = onderwerp.replace('{zaaknr}', this.data.zaak.identificatie);
-        let body = userEventListenerData.zaakOntvankelijk ? this.bodyOntvankelijk : this.bodyNietOntvankelijk;
-        body = body.replace('{zaaktype naam}', this.data.zaak.zaaktype.identificatie).replace('{zaaknr}', this.data.zaak.identificatie);
-
         if (this.sendMailFormField.formControl.value) {
-            const mailObject: MailObject = new MailObject();
-            mailObject.createDocumentFromMail = true;
-            mailObject.onderwerp = onderwerp;
-            mailObject.body = body;
-            mailObject.ontvanger = this.ontvangerFormField.formControl.value;
-            this.mailService.sendMail(this.data.zaak.uuid, mailObject).subscribe(() => {});
+            this.mailtemplateService.findMailtemplate(userEventListenerData.zaakOntvankelijk ?
+                Mail.ZAAK_ONTVANKELIJK : Mail.ZAAK_NIET_ONTVANKELIJK, this.data.zaak.uuid)
+                .subscribe(mailtemplate => {
+                    let onderwerp = mailtemplate.onderwerp;
+                    onderwerp = onderwerp.replace('{zaaknr}', this.data.zaak.identificatie);
```

Comment:
Voor strings als {zaaknr} en {zaaktype naam} komt neem ik aan nog wel een enum of zo. Zodat dit soort betekenisvolle strings niet all over the place terechtkomen. Of is dit slechts een test stukje? Want het resolven van template variabelen zal tochzo veel als mogelijk  in de backend gaan plaatsvinden?

Reply:
Ja dit is oud spul, dit wordt straks op 1 centrale plek gedaan met variabelen uit de enum serverside.